### ISSUE TITLE
[BUG] Broken flake8 integration in v1.5.0

### ISSUE THREAD

**Issue by webknjaz** [terrencepreilly/darglint#107]
After upgrading, flake8 started reporting a lot of violations that are not reported when running it directly. It seems that it no longer respects its own config location when used with flake8

**Comment by terrencepreilly** [comment]
Could you give some more details?  What type of config do you have? What settings?  How are you invoking it?

I tried to reproduce, but it seems to work for me.  I tried having a config file, `.darglint` with the contents

```
[darglint]
strictness=short
```

and a flake8 config file, `.flake8`:

```
strictness=full
```

and a source file, `example.py`:

```
def f(x):
    """Misses x."""
    return x * 2
```

When I run with `darglint example.py`, I get no errors.  When I run with `flake8 example.py`, I get

```
example.py:2:1: DAR201 Missing "Returns" in Docstring: - return
example.py:2:1: DAR101 Missing parameter(s) in Docstring: - x
```

So there must be some configuration setup that I've missed.

**Comment by terrencepreilly** [comment]
Or do you mean that, having no configuration options for flake8, it fails to use the existing darglint config?

**Comment by webknjaz** [comment]
> Or do you mean that, having no configuration options for flake8, it fails to use the existing darglint config?

Exactly. Here's the repo: https://github.com/ansible/pylibssh.

It has the following`.darglint`:
```ini
[darglint]
docstring_style=sphinx
enable=DAR104
strictness=long
```

And no configuration related to darglint in `.flake8` config.

To reproduce:
```console
$ git clone https://github.com/ansible/pylibssh.git
$ cd pylibssh
$ git checkout 2d7b003 --hard  # version where it's definitely broken
$ pip install tox
$ tox -e lint
```

**Comment by Harrison88** [comment]
I think I see how this came up. Currently, flake8 config overrides `.darglint` config. The problem is that the flake8 config has defaults set, so those defaults always override anything in `.darglint`. The solution should be pretty simple. Just change the default in `add_options` to a sentinel value, then in `parse_options` check for that sentinel value and only override `.darglint` if it's not set. I can probably put in a pull request this afternoon or tomorrow, unless you want to go ahead and take care of it @terrencepreilly

**Comment by terrencepreilly** [comment]
Sorry, I didn't see your last comment before fixing this.  I'll push up this fix for v1.5.1 today, once I've run the full integration test suite.  Thanks!

**Comment by terrencepreilly** [comment]
Okay, this ought to have been resolved as of v1.5.1, which I just pushed.

**Comment by webknjaz** [comment]
Thanks! I confirm that it fixed the regression for me.
